dev/AppNotifications/AppNotificationBuilder/AppNotificationComboBox2.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#pragma once
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationComboBox2.g.h"
+
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    struct AppNotificationComboBox2 : AppNotificationComboBox2T<AppNotificationComboBox2, winrt::Windows::Foundation::IStringable>
+    {
+        AppNotificationComboBox2(winrt::hstring const& id);
+
+        // Properties
+        void Title(winrt::hstring const& value) { m_title = value; };
+        winrt::hstring Title() { return m_title; };
+
+        void SelectedItem(winrt::hstring const& value) { m_selectedItem = value; };
+        winrt::hstring SelectedItem() { return m_selectedItem; };
+
+        // Fluent Setters
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox2 AddItem(winrt::hstring const& id, winrt::hstring const& content);
+
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox2 SetTitle(winrt::hstring const& value);
+
+        winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox2 SetSelectedItem(winrt::hstring const& id);
+
+        // IStringable
+        winrt::hstring ToString();
+
+    private:
+        winrt::hstring m_id{};
+        winrt::Windows::Foundation::Collections::IVector<winrt::hstring> m_ids{ winrt::single_threaded_vector<winrt::hstring>() };
```

Comment:
Since the vector and the map aren't exposed as properties in this version of the class, maybe we should ditch the winrt stuff and use STL instead.

Reply:
Resolving as this has become irrelevant. The vector is exposed through a property.